packages/rke2-calico/generated-changes/patch/values.yaml.patch

```diff
@@ -41,3 +41,10 @@
 +
 +global:
 +  systemDefaultRegistry: ""
++
++ipamConfig:
++  strictAffinity: true
++  autoAllocateBlocks: true
```

Comment:
this is actually not needed, it defaults to true any way so can be dropped

Reply:
We need that parameter when defining an ipamConfig:
```
            required:
            - autoAllocateBlocks
            - strictAffinity
```